packages/web/components/complex/transfer.tsx

```diff
@@ -36,6 +36,7 @@ export type TransferProps = {
   warningMessage?: string;
   toggleIsMax: () => void;
   transferFee?: CoinPretty;
+  transferPathHealth?: "Healthy" | "Congested" | "Blocked" | "Undefined";
```

Comment:
Wouldn't we only need congested or blocked? The question mark next to the member means it doesn't need to be there, which is the undefined case as well. That handles all other cases, including when it's not blocked.

Reply:
Yeah that makes sense, I'll update to just `Congested` | `Blocked`